src/components/header.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const Header = () => (
+  <nav id="nav">
+    <h1 className="logo">
+      Bookstore CMS
+    </h1>
+    <ul className="navlinks">
+      <li>
+        <a href="/">Books</a>
+      </li>
+      <li>
+        <a href="/categories">Categories</a>
+      </li>
+    </ul>
```

Comment:
Kindly consider using the react Link tag, <Link>, instead of the a tag since it gives you a  lot of benefits like active class styling or browsing history thereby making life easier and simple.